### PR TITLE
11238 ga events

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11238)

This PR is to add Google Analytics events to the wizard for the chapter 36 form. All events were tested using Dataslayer in Firefox and Google tag assistant in Chrome and are firing correctly.

## Screenshots
Here are screenshots of the custom events we added, the rest of the events come with the platform.

<details><summary>howToWizard-formChange</summary>

Screen Shot 2020-09-24 at 12 55 30 PM

Screen Shot 2020-09-24 at 12 56 01 PM

Screen Shot 2020-09-24 at 12 56 39 PM

Screen Shot 2020-09-24 at 12 57 32 PM

Screen Shot 2020-09-24 at 12 57 58 PM

Screen Shot 2020-09-24 at 12 58 21 PM

Screen Shot 2020-09-24 at 12 59 00 PM

Screen Shot 2020-09-24 at 12 59 11 PM

Screen Shot 2020-09-24 at 12 59 39 PM

Screen Shot 2020-09-24 at 12 59 48 PM

</details>

<details><summary>howToWizard-notice-displayed</summary>

Screen Shot 2020-09-24 at 1 01 17 PM

Screen Shot 2020-09-24 at 1 05 02 PM

Screen Shot 2020-09-24 at 1 02 30 PM

Screen Shot 2020-09-24 at 1 02 22 PM

</details>

## Acceptance criteria
- [x] Code/event names provided by the GA/Insights Team has been inserted into our feature for tracking

### CODE REVIEW

File: src/applications/vre/28-8832/wizard/pages/IneligibleNotice.jsx [hunk 1]
```diff
@@ -8,6 +9,10 @@ import {
 const IneligibleNotice = () => {
   useEffect(() => {
     sessionStorage.setItem(WIZARD_STATUS, WIZARD_STATUS_INELIGIBLE);
+    recordEvent({
+      event: `howToWizard-notice-displayed`,
+      'reason-for-notice': 'ineligibility - not a service member or veteran',
```
Comment (micahchiang) [inline src/applications/vre/28-8832/wizard/pages/IneligibleNotice.jsx:14]: Can we include dependent in here as well? So something like `ineligibility  not a service member or veteran, or a dependent of one`

**Review by micahchiang — APPROVED** [review]
LGTM just left a comment on possibly updating the reason for notice on `IneligibleNotice`

File: src/applications/vre/28-8832/wizard/pages/VRECounselorNotification.jsx [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+  useEffect(() => {
+    recordEvent({
+      event: `howToWizard-notice-displayed`,
+      'reason-for-notice': 'ineligibility - eligible for chapter 31',
+    });
+  });
+  return (
+    <div className="vads-u-margin-top--2 vads-u-background-color--primary-alt-lightest vads-u-padding--3">
+      <p className="vads-u-margin-top--0">
+        Please contact your vocational rehabilitation counselor to learn more
```
Comment (Mottie) [inline src/applications/vre/28-8832/wizard/pages/VRECounselorNotification.jsx:15]: I thought VRE was renamed to Veteran Readiness and Employment... so are the counselor's now named "Veteran Readiness counselor"?

File: src/applications/vre/28-8832/wizard/pages/IneligibleNotice.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/vre/28-8832/wizard/pages/IneligibleNotice.jsx:14]: These were set up by the analytics team, I guess they have a system for how they assign these

**Pull request merged**